apps/admin_app/lib/admin_app_web/views/order_view.ex

```diff
@@ -234,4 +238,12 @@ defmodule AdminAppWeb.OrderView do
     {:ok, total} = Money.to_string(OrderDomain.total_amount(order))
     total
   end
+
+  def get_support_url() do
+    System.get_env("SUPPORT_URL")
```

Comment:
I think we should set the keys for this in application config and then use `Application.get_env` to fetch this. We will have to do this because once general settings feature is merged the name of website and email address would be set by the admin there and it should be picked from there.

Reply:
done